experimenter/experimenter/experiments/tests/api/v5/test_serializers/test_nimbus_ready_for_review_serializer.py

```diff
@@ -1257,61 +1257,111 @@ def test_no_treatment_branches_for_rollout(self):
             [NimbusConstants.ERROR_SINGLE_BRANCH_FOR_ROLLOUT],
         )
 
-    def test_rollout_valid_version_support(self):
-        desktop = NimbusExperiment.Application.DESKTOP
+    @parameterized.expand(
+        [
+            (
+                NimbusExperiment.Application.DESKTOP,
+                NimbusExperiment.Version.FIREFOX_105,
+            ),
+            (
+                NimbusExperiment.Application.FENIX,
+                NimbusExperiment.Version.FIREFOX_105,
+            ),
+            (
+                NimbusExperiment.Application.FOCUS_ANDROID,
+                NimbusExperiment.Version.FIREFOX_105,
+            ),
+            (
+                NimbusExperiment.Application.IOS,
+                NimbusExperiment.Version.FIREFOX_105,
+            ),
+            (
+                NimbusExperiment.Application.FOCUS_IOS,
+                NimbusExperiment.Version.FIREFOX_105,
+            ),
+        ]
```

Comment:
I get your point, but I am saying it's not getting passed in parameters, you are only mentioning application, version here, targeting config is missing

Reply:
and ci also refers to targeting is missing in parameters

Reply:
one issue is now for all you have added `TargetingConfig.MAC_ONLY`